src/lib/form-field/form-field.md

```diff
@@ -8,6 +8,7 @@ In this document, "form field" refers to the wrapper component `<mat-form-field>
 
 The following Angular Material components are designed to work inside a `<mat-form-field>`:
 * [`<input matInput>` &amp; `<textarea matInput>`](https://material.angular.io/components/input/overview)
+* [`<select matNativeControl>`](https://material.angular.io/components/select/overview)
```

Comment:
Should also update the line above:
`<input matNativeControl>` and `<textarea matNativeControl>`